src/Application/Application.csproj

```diff
@@ -2,6 +2,8 @@
 
   <PropertyGroup>
     <TargetFramework>net7.0</TargetFramework>
+    <RootNamespace>CleanArchitecture.Application</RootNamespace>
```

Comment:
```suggestion
    <RootNamespace>SSW.CleanArchitecture.Application</RootNamespace>
```

Reply:
I'm not sure if this matters, as `CleanArchitecture` will be replaced by the directory name that the solution is installed into.  We could make this change, but it would be superficial.

Reply:
@william-liebenberg as per our conversation, I'll update so that the namespace and assembly name reflect the SSW branding.  The project names will remain as is (i.e. `Application`, `Domain`, etc.).